services/web-app/pages/about-carbon/releases.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { NextSeo } from 'next-seo'
+
+import RemoteMdxLoader from '@/components/remote-mdx-loader'
+import { getRemoteMdxData } from '@/lib/github'
+
+const RemoteMdxPage = ({ source }) => {
```

Comment:
Link: Should 'V11 release page' link to http://localhost:3000/migrating/guide/overview or somewhere else?

Reply:
Good catch, this is where its linking to now. https://carbondesignsystem.com/migrating/guide/overview/
where will this content be on the new site?